docs/requests.md

```diff
@@ -61,7 +61,7 @@ For example: `request.path_params['username']`
 #### Client Address
 
 The client's remote address is exposed as a named two-tuple `request.client`.
```

Comment:
I think the sentence got too complex below... Can we just do this, or something?
```suggestion
The client's remote address is exposed as a named two-tuple `request.client` (or `None`).
```